src/vs/workbench/contrib/webview/browser/pre/index.html

```diff
@@ -236,7 +236,7 @@
 				return reject(new Error('Service Workers are not enabled. Webviews will not work. Try disabling private/incognito mode.'));
 			}
 
-			const swPath = encodeURI(`service-worker.js?v=${expectedWorkerVersion}&vscode-resource-base-authority=${searchParams.get('vscode-resource-base-authority')}&remoteAuthority=${searchParams.get('remoteAuthority') ?? ''}`);
```

Comment:
Can you also please apply this in the `index-no-csp.html` file. That file is a duplicate of this one but without the content security policy